commands-yml/commands/device/clipboard/get-clipboard.yml

```diff
@@ -8,6 +8,7 @@ description:
 
     (For iOS real devices)
     Apple security preferences require the WebDriverAgentRunner application to be in [foreground](https://github.com/appium/WebDriverAgent/issues/330) in order to be able to receive the system clipboard content. Otherwise an empty string is always returned. Consider using [Activate App](/docs/en/commands/device/app/activate-app.md) and [Background App](/docs/en/commands/device/app/background-app.md) to change the foreground application.
+    Activate App command cannot launch the WebDriverAgent properly on some environment. It depends on XCTest framework by Apple. Then, you should launch the WebDriverAgent from the app icon. For example, you can achive it with opening spring board with `@driver.activate_app 'com.apple.springboard'`, finding the WDA icon with `@driver.find_element :accessibility_id, 'WebDriverAgentRunner-Runner'` and click it and calling get clipboard API.
```

Comment:
on some environments
launch WebDriverAgent using its Springboard's application icon

The last sentence I would rather simplify by changing it structure to a bulleted/numbered list